src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -169,14 +169,24 @@ def respond_to_slack_command(
 ) -> None:
     log = "slack.link-identity." if command == "link" else "slack.unlink-identity."
 
-    # TODO: ignore expired url errors
     if params.response_url:
         logger.info(log + "respond-webhook", extra={"response_url": params.response_url})
         try:
             webhook_client = WebhookClient(params.response_url)
             webhook_client.send(text=text, replace_original=False, response_type="ephemeral")
+            metrics.incr(
+                SLACK_COMMANDS_LINK_IDENTITY_SUCCESS_DATADOG_METRIC,
+                sample_rate=1.0,
+                tags={"type": "webhook", "command": command},
+            )
         except (SlackApiError, SlackRequestError) as e:
-            logger.exception(log + "error", extra={"error": str(e)})
+            if "Expired url" not in str(e):
```

Comment:
i wish the sdk had more specific error types

Reply:
We should spike on building a utility around this. I feel like our own list of expected error text (perhaps an enum class), and a common function for parsing them out, would go a long way.